cibuild.cmd

```diff
@@ -58,14 +59,19 @@ if /i "%HOST%"=="CoreCLR" (
     goto :error
 )
 
+set LOCALIZED_BUILD_ARGUMENT=
+if "%LOCALIZED_BUILD%"=="true" (
+    set LOCALIZED_BUILD_ARGUMENT="/p:LocalizedBuild=true"
+)
+
 :: Restore build tools
 call %~dp0init-tools.cmd
 
 echo.
 echo ** Rebuilding MSBuild with downloaded binaries
 
 set MSBUILDLOGPATH=%~dp0msbuild_bootstrap_build.log
-call "%~dp0build.cmd" /t:Rebuild /p:Configuration=%BUILD_CONFIGURATION% /p:"SkipBuildPackages=true"
+call "%~dp0build.cmd" /t:Rebuild /p:Configuration=%BUILD_CONFIGURATION% /p:"SkipBuildPackages=true" %LOCALIZED_BUILD_ARGUMENT%
```

Comment:
However, you do get dirty xlf files each time. Okay, my next thing is to separate xlf synchronization, just for the experience of it :)